### PR TITLE
DR6 updates to website

### PR DESCRIPTION
**Pull request by geordie666**
Updated to release version for DR6. Includes:

- Latest data model
- Latest status plots
- Added histogram plots for depth instead of referring back to DR2 calculations
- General updates to the description page for area/number of sources, etc.
- Links to NOAO's publications page for the LS in addition to updating publications from the LS team

### CODE REVIEW

**Comment by geordie666** [comment]
@weaverba137: Let me just update the CHANGES.md file, too, before you merge this

**Comment by weaverba137** [comment]
Looking at this now...

File: CHANGES.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
 # legacysurvey Change Log
 
-## 6.0.0 (DR6, unreleased)
+## 6.0.0 (DR6, )
```
Comment (weaverba137) [inline CHANGES.md]: I think it is safe to put today's date in here.

File: pages/release.rst [hunk 2]
```diff
@@ -37,3 +36,15 @@ RELEASE	CAMERA FILTER     PRIMARY
 5000	wise   :math:`W3` 0
 5000	wise   :math:`W4` 0
 ======= ====== ========== =======
+
+======= ====== ========== =======
+RELEASE	CAMERA FILTER     PRIMARY
```
Comment (weaverba137) [inline pages/release.rst]: The spacing in this table looks a bit weird.  Double check that that is not affecting the display of the table.

File: CHANGES.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # legacysurvey Change Log
 
-## 6.0.0 (DR6, )
+## 6.0.0 (DR6, 2018-2-23)
```
Comment (weaverba137) [inline CHANGES.md]: One more minor thing: two-digit month please, for consistency with the rest of the file.

**Comment by weaverba137** [comment]
Minor comment about the date format, but otherwise this is ready.

**Comment by geordie666** [comment]
Thanks for the review. Good catch on the ```RELEASE``` table. I did a little aliasing and it seems to render in a nice, compact form now:

https://github.com/geordie666/legacysurvey/blob/2b477512a78bab3f9a085108861ff1d35b7ceb1c/pages/release.rst

**Comment by geordie666** [comment]
Good point on the date, too. Should now be in MM format.

**Comment by weaverba137** [comment]
Merged.  I'll tag momentarily.

**Pull request merged**